releng/unit-tests/fs/usr/local/bin/influxdb_prebuild_tests.bash

```diff
@@ -1,7 +1,12 @@
-#!/bin/bash
+#!/bin/bash -e
```

Comment:
Are there any other scripts missing the `-e`?

Reply:
I'm updating them all as I touch them. Lots were initially missing them.